packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.tsx

```diff
@@ -140,9 +157,47 @@ export class LinkerOverlayContent extends Component<
   }
 
   handleMouseMove(event: MouseEvent): void {
```

Comment:
My bad, you still actually need to handle a window resize.

Positioning the dialog near the top edge and resizing shorter, now makes it go off screen at the top, rather than bottom :(

bottom right positioning is still better, as now it keeps position better though.